stable/datadog/values.yaml

```diff
@@ -274,6 +274,30 @@ datadog:
 #      cpu: 200m
 #      memory: 256Mi
 
+## @param systemProbe - object - required
+## Enable systemProbe agent and provide custom configs
+#
+systemProbe:
+  ## @param enabled - boolean - required
+  ## Set this to true to enable system-probe agent
+  #
+  enabled: false
+  ## @param debugPort - integer - required
+  ## Specify the port to expose pprof and expvar for system-probe agent
+  #
+  debugPort: 0
+
+  ## @param enableConntrack - boolean - required
+  ## Enable the system-probe agent to connect to the netlink/conntrack subsystem to add NAT information to connection data
+  ## Ref: http://conntrack-tools.netfilter.org/
+  #
+  enableConntrack: true
+
+  ## @param bpfDebug - boolean - optional
+  ## Enable logging for kernel debug
+  #
+  # bpfDebug: false
```

Comment:
why is `bpfDebug` commented out, but everything else isn't?

Reply:
Because `bpfDebug` is optional.